pkg/webreg/webreg.go

```diff
@@ -328,6 +333,195 @@ const templateDefinitions = `
+allows users to create new tests by combining smaller, individual test steps.
+These individual tests can be put into a shared registry that other tests can
+access. This results in test workflows that are easier to maintain and
+upgrade as multiple test workflows can share steps and don’t have to each be
+updated individually to fix bugs or add new features. It also reduces the
+chances of a mistake when copying a feature from one test workflow to
+another.
+</p>
+
+<p>
+To understand how the multistage tests and registry work, we must first talk
+about the 3 components of the test registry and how to use those components
+to create a test.
+</p>
+
+<h2>Reference:</h2>
+<p>
+A reference is the lowest level component in the test step registry. A
+reference defines a base container image for a step, the filename of the
+shell script to run inside the container, the resource requests and limits
+for the container, and documentation for the reference. Example of a
+reference:
+</p>
+
+<pre>
+ref:
+  as: ipi-conf
+  from: centos:7
+  commands: ipi-conf-commands.sh
+  resources:
+    requests:
+      cpu: 1000m
+      memory: 100Mi
+  documentation: |-
+	The IPI configure step generates the install-config.yaml file based on the cluster profile and optional input files.
+</pre>
+
+<p>
+Note: the shell script file must follow the naming convention described later
+in this help page.
+</p>
+
+<p>
+The commands file must contain shell script in a shell language supported by
+the <code>shellcheck</code> program used to validate the commands. However,
+regardless of the shell language used for the commands, the web UI will
+syntax highlight all commands as bash.
+</p>
+
+<p>
+Sharing files between steps is supported via a shared directory. All
+containers will have an environment variable <code>SHARED_DIR</code> which
+contains the path of the shared directory. An artifacts directory also exists
+for steps that produce artifacts. The path for the artifacts directory is
+stored in the <code>ARTIFACTS_DIR</code> environment variable.
+</p>
+
+<p>
+A reference may be referred to in chains, workflows, and ci-operator configs.
+</p>
+
+<h2>Chain:</h2>
+<p>
+A chain is a registry component that specifies multiple steps to be run.
+Steps are run in the order that they are written. Steps specified by a chain
+can be either references and other chains. Example of a chain:
+</p>
+
+<pre>
+chain:
+  as: ipi-deprovision
+  steps:
+  - chain: gather
+  - ref: ipi-deprovision-deprovision
+  documentation: |-
+    The IPI deprovision step chain contains all the individual steps necessary to deprovision an OpenShift cluster.
+</pre>
+
+<h2>Workflow:</h2>
+<p>
+A workflow is the highest level component of the step registry. It is almost
+identical to the syntax of the ci-operator config for multistage tests and
+defines an entire test from start to finish. It has 4 basic components: a
+cluster_type string (eg: <code>aws</code>, <code>azure4</code>,
```

Comment:
Should be `cluster_profille` and IMHO in code blocks. Is `s/azure4/azure/` I think?

Reply:
Yup, will change to `cluster_profile`. For azure, the profile is `azure4`: https://github.com/openshift/ci-tools/blob/master/pkg/api/types.go#L511